<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationParameterDialog</class>
 <widget class="QDialog" name="StationParameterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Station</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>553</x>
     <y>20</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit:Per Unit</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_parameter">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>72</y>
     <width>671</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ref-Voltage</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Voltage</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Angle</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Energy</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Load</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Compensation</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>270</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>113</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Station Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>137</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_voltage">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>270</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tideway">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Power flow...</string>
   </property>
  </widget>
  <zorder>pushButton_voltage</zorder>
  <zorder>pushButton_tideway</zorder>
  <zorder>tableWidget_parameter</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_OK</zorder>
  <zorder>line</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
